app/utils/remix.server.ts

```diff
@@ -0,0 +1,103 @@
+import { type CacheControlValue, parse, format } from '@tusbar/cache-control'
```

Comment:
I think calling this file `remix.server.ts` may not be the right call anymore. Maybe this should just be called `headers.server.ts`.

Also, could you add some jsdoc comments to these utilities? Thanks!